cmd/app/use.go

```diff
@@ -0,0 +1,108 @@
+package app
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
```

Comment:
I guess name and email are missing here :)

Reply:
Upsss....;) 
it goes my or stormkit ones?

Reply:
I would keep a LICENSE file similar to https://github.com/stormkit-io/app-stormkit-io/blob/master/LICENSE and link it to the README

Reply:
Done in [70f6afc](https://github.com/stormkit-io/stormkit-cli/pull/1/commits/70f6afc1a6172318b4378eca77ed3fb25c65b5c5)